include/core-image-renesas-base.inc

```diff
@@ -39,3 +39,164 @@ IMAGE_BOOT_FILES:rzg2l-sbc = " \
 "
 
 DEPENDS += " linux-yocto uenv"
+
+IMAGE_LINK_NAME = "${IMAGE_BASENAME}${IMAGE_MACHINE_SUFFIX}"
+
+add_bootloader_rootfs() {
+    install -d ${IMAGE_ROOTFS}/boot/uload-bootloader
+    cp -rf ${DEPLOY_DIR_IMAGE}/target/images/bl2_bp-rzg2l-sbc.bin ${IMAGE_ROOTFS}/boot/uload-bootloader
+    cp -rf ${DEPLOY_DIR_IMAGE}/target/images/fip-rzg2l-sbc.bin ${IMAGE_ROOTFS}/boot/uload-bootloader
+}
+
+deploy_package() {
+    rm -f ${DEPLOY_DIR_IMAGE}/bl31*
+    rm -f ${DEPLOY_DIR_IMAGE}/*_esd-*
+    rm -f ${DEPLOY_DIR_IMAGE}/*pmic*
+    rm -f ${DEPLOY_DIR_IMAGE}/*PMIC*
+    rm -f ${DEPLOY_DIR_IMAGE}/*.elf
+    rm -f ${DEPLOY_DIR_IMAGE}/Image-rzpi*
+    rm -f ${DEPLOY_DIR_IMAGE}/rzpi-rzpi*
+    rm -f ${DEPLOY_DIR_IMAGE}/modules*
+    rm -f ${DEPLOY_DIR_IMAGE}/u-boot*
+}
+
+# Clean the output directory after the build
+do_release_clean() {    
+    rm -f ${DEPLOY_DIR_IMAGE}/Image*
+    rm -f ${DEPLOY_DIR_IMAGE}/rzpi*
+    rm -f ${DEPLOY_DIR_IMAGE}/${IMAGE_BASENAME}*
+    rm -f ${DEPLOY_DIR_IMAGE}/fip-rzpi*
+    rm -f ${DEPLOY_DIR_IMAGE}/bl2*
+    rm -f ${DEPLOY_DIR_IMAGE}/core-image*
+    rm -f ${DEPLOY_DIR_IMAGE}/Readme.md
+    rm -f ${DEPLOY_DIR_IMAGE}/target/images/*PMIC*
+}
```

Comment:
These two functions are supposed to clean the output directory, so there is no reason to separate them into two functions.

Also,please make sure there are no redundant remove commands. Recheck the build output.

Reply:
Updated in the commit: fa4a70cd5eb6037a6490416b3944b4b8ce0fae77